Source/WTF/wtf/PlatformHave.h

```diff
@@ -1018,11 +1018,7 @@
 #define HAVE_PER_APP_ACCESSIBILITY_PREFERENCES 1
 #endif
 
-#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110000) \
-    || (PLATFORM(MACCATALYST) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 140000) \
-    || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 140000) \
-    || (PLATFORM(APPLETV) && __TV_OS_VERSION_MAX_ALLOWED >= 140000) \
-    || (PLATFORM(WATCHOS) && __WATCH_OS_VERSION_MIN_REQUIRED >= 70000)
+#if PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000
```

Comment:
This doesn't look right to me, do you want `#if !PLATFORM(MAC) || __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000`?

Reply:
You're right that this is probably better.